cmd/limactl/list.go

```diff
@@ -155,7 +159,22 @@ func listAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	return store.PrintInstances(cmd.OutOrStdout(), instances, format)
+	allFields, err := cmd.Flags().GetBool("all-fields")
+	if err != nil {
+		return err
+	}
+
+	options := store.PrintOptions{AllFields: allFields}
+	out := cmd.OutOrStdout()
+	if out == os.Stdout {
+		if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {
+			options.IsTerminal = true
```

Comment:
The `options.IsTerminal` field is not really used for anything (because it being `false` is treated the same way as `TerminalWidth` being `0`), so I would drop it.